convert2rhel/toolopts.py

```diff
@@ -263,9 +288,20 @@ def _process_cli_options(self):
         parsed_opts, _ = self._parser.parse_args()
 
         global tool_opts  # pylint: disable=C0103
+
         if parsed_opts.debug:
```

Comment:
I had to move parsing the debug option, because without that is impossible to use debug logging in `options_from_config_files`